Package.swift

```diff
@@ -18,7 +18,10 @@ let package = Package(
             revision: "d8003787efafa82f9805594bc51100be29ac6903"), // on release/1.1
         .package(
             url: "https://github.com/apple/swift-foundation-icu",
-            revision: "0c1de7149a39a9ff82d4db66234dec587b30a3ad")
+            revision: "0c1de7149a39a9ff82d4db66234dec587b30a3ad"),
+        .package(
+            url: "https://github.com/apple/swift-format.git",
+            branch: "main")
```

Comment:
Do we want to tie ourselves to an unstable branch here, or should we add a release-based dependency?